lib/evmone/eof.hpp

```diff
@@ -69,6 +69,7 @@ enum class EOFValidationError
     header_terminator_missing,
     type_section_missing,
     code_section_missing,
+    unknown_section_id,
```

Comment:
Why is this added?

Reply:
I remove it in the 2nd commit

Reply:
I think you should remove this change from the first commit.

Reply:
1st commit doesn't work right without it, I wanted to squash them, if second approach is okay.

Reply:
I don't see where this is used in the first commit. But the second approach is ok.